src/components/sykmeldingpanel/AnnenInfo.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Information } from '@navikt/ds-icons'
+
+import { IconHeading } from '../shared/IconHeading/IconHeading'
+import { SykmeldingFragment } from '../../graphql/queries/graphql.generated'
+import { formatDate } from '../../utils/dateUtils'
+import { cleanId } from '../../utils/stringUtils'
+import { isUtenlandsk } from '../../utils/utenlanskUtils'
+import { ListItem } from '../shared/listItem/ListItem'
+
+import styles from './AnnenInfo.module.css'
+
+interface Props {
+    sykmelding: SykmeldingFragment
+}
+
+function AnnenInfo({ sykmelding }: Props): JSX.Element {
+    const title = 'Annen info'
+    return (
+        <li aria-labelledby={cleanId(title)}>
+            <IconHeading title={title} Icon={Information} />
+            <ul className={styles.annenInfo}>
+                <ListItem
+                    title="Dato sykmeldingen ble skrevet"
+                    text={formatDate(sykmelding.behandletTidspunkt)}
+                    headingLevel="4"
+                    blueListItem
+                />
+
+                {!isUtenlandsk(sykmelding) ? (
+                    <>
+                        <ListItem
+                            title="Sykmeldingen ble skrevet av"
+                            text={[
+                                sykmelding.behandler?.navn ?? 'Ukjent',
+                                sykmelding.behandler?.telefon ? `Tlf: ${sykmelding.behandler.telefon}` : 'Tlf: —',
+                            ]}
+                            headingLevel="4"
+                            blueListItem
+                        />
+                        {sykmelding.arbeidsgiver && sykmelding.arbeidsgiver.navn && (
```

Comment:
Flisespikkeri, men null-coalescing-operatoren lar oss gjøre dette, sparer oss litt `&&`s :smile: 

```suggestion
                        {sykmelding.arbeidsgiver?.navn && (
```